src/scripts/main.js

```diff
@@ -354,7 +354,42 @@ const people = [
   },
 ];
 
-// eslint-disable-next-line no-console
-console.log(people); // you can remove it
+const table = document.querySelector('.dashboard');
```

Comment:
This is a potential performance issue. Each time you append a row to the table, the browser has to recalculate the layout, repaint the screen, and perform other tasks that can slow down your application. It's more efficient to create all the rows first, append them to a document fragment, and then append that fragment to the table. This way, the browser only has to update the layout once.